src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+function copyTabCol() {
+  const table = document.querySelector('table');
+  const rows = table.querySelectorAll('tr');
+
+  rows.forEach((row) => {
+    const cellToCopy = row.querySelector(':nth-child(2)');
```

Comment:
Ensure that `cellToCopy` is not `null`. If a row does not have a second child, `querySelector(':nth-child(2)')` will return `null`, leading to errors in subsequent lines.